src/unperturbed_perturbed_comparison.py

```diff
@@ -168,11 +170,12 @@
     (perturbed_gait_data.gait_cycles[:, :, 'RHEE.PosZ'] -
      perturbed_gait_data.gait_cycles[:, :, 'LHEE.PosZ']).values.flatten()
 
-sbn.boxplot([unperturbed_stride_widths, perturbed_stride_widths],
+sbn.boxplot(data=[unperturbed_stride_widths, perturbed_stride_widths],
             ax=axes[-1],
-            color=[red, blue],
+            #color=[red, blue],
             whis='range',
-            names=['U', 'P'])
+            #x=['U', 'P'],
+            )
```

Comment:
The API for seaborn has changed significantly and I can't seem to install an old enough version of Seaborn.